store-conf/conf/log4j.properties.production

```diff
@@ -103,6 +103,14 @@ log4j.appender.EWS.File=/opt/zimbra/log/ews.log
 log4j.appender.EWS.layout=com.zimbra.common.util.ZimbraPatternLayout
 log4j.appender.EWS.layout.ConversionPattern=%d %-5p [%t] [%z] %c{1} - %m%n
 
+#Appender EVENTS writes to the file "event.log".
+log4j.appender.EVENTS=org.apache.log4j.rolling.RollingFileAppender
+log4j.appender.EVENTS.RollingPolicy=org.apache.log4j.rolling.TimeBasedRollingPolicy
+log4j.appender.EVENTS.RollingPolicy.FileNamePattern=/opt/zimbra/log/event.log.%d{yyyy-MM-dd}
+log4j.appender.EVENTS.File=/opt/zimbra/log/event.log
+log4j.appender.EVENTS.layout=com.zimbra.common.util.ZimbraPatternLayout
+log4j.appender.EVENTS.layout.ConversionPattern=%m%n
```

Comment:
Please add a comment explaining why this conversion pattern differs from the ones for most other appenders.
If nothing else, may save bad future copy/pastes.

Reply:
Good idea. I changed the conversion pattern to simplify the log output and to potentially make it easier to parse the logs back into Event objects in the future.